src/main.rs

```diff
@@ -204,7 +204,7 @@ fn find_variables(val: &hcl::Value) -> Vec<String> {
             let tmpl = hcl::Template::from_str(&s).unwrap();
```

Comment:
intentional `unwrap()`?

Reply:
yes, there are still a handful of unhandled panics about